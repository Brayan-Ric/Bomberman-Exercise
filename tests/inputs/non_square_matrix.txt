R _ W
B2 _ F1