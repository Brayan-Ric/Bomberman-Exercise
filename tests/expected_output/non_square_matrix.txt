Error: El tablero no es cuadrado